settings.py

```diff
@@ -15,6 +15,7 @@
 # These apps are great during development.
 INSTALLED_APPS += (
     'django_extensions',
+    'djcelery',
```

Comment:
This is deprecated, and isn't necessary since we upgraded to Celery 3.1. Celery commands run via `celery -A olympia ...` now.